src/main/scala/io/sdkman/changelogs/Migrations.scala

```diff
@@ -1,6 +1,6 @@
 package io.sdkman.changelogs
 
-import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.github.mongobee.changeset.{ ChangeLog, ChangeSet }
```

Comment:
Not related to the PR, and not the standard scala code style.

Reply:
Sorry not a scala dev and just downloaded the scala ide for the pull requests. Do you have some form of setting for the scala ide so i have the right formatting?